lib/text.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+
+class Text
+  def initialize
+    account_sid = 'AC5e4e1ddec6330a437df6f9f1345820a4'
```

Comment:
Will want to be careful as I can see your unique codes on a public platform! May want to keep these hidden using environment variables!